cmd/plugin/cli/cmd/info.go

```diff
@@ -0,0 +1,110 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/tsuru/rpaas-operator/build/cli/proxy"
+	"github.com/olekukonko/tablewriter"
+
+)
+
+func init() {
+	rootCmd.AddCommand(infoCmd)
+
+	// Cobra supports local flags which will only run when this command
+	// is called directly, e.g.:
+	infoCmd.Flags().StringP("service", "s", "", "Service name")
+	infoCmd.Flags().StringP("instance", "i", "", "Service instance name")
+	infoCmd.MarkFlagRequired("service")
+	infoCmd.MarkFlagRequired("instance")
+}
+
+// infoCmd represents the info command
+var infoCmd = &cobra.Command{
+	Use:   "info",
+	Short: "Lists avaiable plans and flavors for the specified rpaas-instance",
+	Long:  `Lists avaiable plans and flavors for the specified rpaas-instance`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.ParseFlags(args)
+		info := infoArgs{}
+		info.service = cmd.Flag("service").Value.String()
+		info.instance = cmd.Flag("instance").Value.String()
+		info.prox = &proxy.Proxy{ServiceName: info.service, InstanceName: info.instance, Method: "GET"}
```

Comment:
It's better implement a function `New` in the package `proxy` to instantiate a new object of Proxy.